diff -up smbldap-tools-0.9.9/smbldap.conf.1~ smbldap-tools-0.9.9/smbldap.conf
--- smbldap-tools-0.9.9/smbldap.conf.1~	2012-08-07 13:12:06.000000000 +0200
+++ smbldap-tools-0.9.9/smbldap.conf	2018-07-12 10:22:13.641786347 +0200
@@ -38,7 +38,7 @@ SID="S-1-5-21-2252255531-4061614174-2474
 # Domain name the Samba server is in charged.
 # If not defined, parameter is taking from smb.conf configuration file
 # Ex: sambaDomain="IDEALX-NT"
-sambaDomain="DOMSMB"
+sambaDomain="example"
 
 ##############################################################################
 #
@@ -76,15 +76,15 @@ verify="require"
 
 # CA certificate
 # see "man Net::LDAP" in start_tls section for more details
-cafile="/etc/smbldap-tools/ca.pem"
+cafile="/etc/pki/tls/cacert.pem"
 
 # certificate to use to connect to the ldap server
 # see "man Net::LDAP" in start_tls section for more details
-clientcert="/etc/smbldap-tools/smbldap-tools.example.com.pem"
+clientcert=""
 
 # key certificate to use to connect to the ldap server
 # see "man Net::LDAP" in start_tls section for more details
-clientkey="/etc/smbldap-tools/smbldap-tools.example.com.key"
+clientkey=""
 
 # LDAP Suffix
 # Ex: suffix=dc=IDEALX,dc=ORG
@@ -93,17 +93,17 @@ suffix="dc=example,dc=com"
 # Where are stored Users
 # Ex: usersdn="ou=Users,dc=IDEALX,dc=ORG"
 # Warning: if 'suffix' is not set here, you must set the full dn for usersdn
-usersdn="ou=Users,${suffix}"
+usersdn="ou=People,${suffix}"
 
 # Where are stored Computers
 # Ex: computersdn="ou=Computers,dc=IDEALX,dc=ORG"
 # Warning: if 'suffix' is not set here, you must set the full dn for computersdn
-computersdn="ou=Computers,${suffix}"
+computersdn="ou=Hosts,${suffix}"
 
 # Where are stored Groups
 # Ex: groupsdn="ou=Groups,dc=IDEALX,dc=ORG"
 # Warning: if 'suffix' is not set here, you must set the full dn for groupsdn
-groupsdn="ou=Groups,${suffix}"
+groupsdn="ou=Group,${suffix}"
 
 # Where are stored Idmap entries (used if samba is a domain member server)
 # Ex: idmapdn="ou=Idmap,dc=IDEALX,dc=ORG"
@@ -176,24 +176,24 @@ defaultMaxPasswordAge="45"
 # Just set it to a null string if you want to use the smb.conf 'logon home'
 # directive and/or disable roaming profiles
 # Ex: userSmbHome="\\PDC-SMB3\%U"
-userSmbHome="\\PDC-SRV\%U"
+userSmbHome=""
 
 # The UNC path to profiles locations (%U username substitution)
 # Just set it to a null string if you want to use the smb.conf 'logon path'
 # directive and/or disable roaming profiles
 # Ex: userProfile="\\PDC-SMB3\profiles\%U"
-userProfile="\\PDC-SRV\profiles\%U"
+userProfile=""
 
 # The default Home Drive Letter mapping
 # (will be automatically mapped at logon time if home directory exist)
 # Ex: userHomeDrive="H:"
-userHomeDrive="H:"
+userHomeDrive=""
 
 # The default user netlogon script name (%U username substitution)
 # if not used, will be automatically username.cmd
 # make sure script file is edited under dos
 # Ex: userScript="startup.cmd" # make sure script file is edited under dos
-userScript="logon.bat"
+userScript=""
 
 # Domain appended to the users "mail"-attribute
 # when smbldap-useradd -M is used
